### ISSUE TITLE
"createEventWithOptions" with attribute "recurrence" will cause immediately shut down by Android 4.4.2

### ISSUE THREAD

**Issue by bau720123** [EddyVerbruggen/Calendar-PhoneGap-Plugin#218]
hi @EddyVerbruggen 
I am using PGB 5.2.0 with your latest version from npm
https://www.npmjs.com/package/cordova-plugin-calendar

this is my full example
```
var title = "title";
var newTitle = "newtitle";
var location = "location"
var notes = "notes";
var startDate = new Date(new Date().getTime());
var endDate = new Date(new Date().getTime() + 3*24*60*60*1000);
var futureDate = new Date(new Date().getTime() + 6*24*60*60*1000);
var calendarName = "mycalendar";

var calOptions = window.plugins.calendar.getCalendarOptions();
calOptions.recurrence = "monthly";
window.plugins.calendar.createEventWithOptions(title, location, notes, startDate, endDate, calOptions, success, error);
```

when in Android 4.4.2，it will cause immediately shut down
but when in iOS 8.4.1 (12H321)，no problem at all

finally I tried many times and know what the problem is in Android，just don't use the "recurrence" attribute，just like the code
```
var calOptions = window.plugins.calendar.getCalendarOptions();
window.plugins.calendar.createEventWithOptions(title, location, notes, startDate, endDate, calOptions, success, error);
```

but don't know why ?

**Comment by EddyVerbruggen** [comment]
I think it's the same cause as #216, so please try again with the newer release (about to build it).

**Comment by bau720123** [comment]
problem solved in 4.4.0 version
thanks @EddyVerbruggen
